spec/models/spree/retail/shopify/generate_refund_order_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+Spree.describe Spree::Retail::Shopify::GenerateRefundOrder, type: :model do
+  include_context 'shopify_request'
+
+  let!(:store) { create :store }
```

Comment:
I don't feel confortable in refactoring it, since it will require a lot of work.

We would also need to refactor how the similar order class work, because why do one and not do the other?

I feel comfortable with the object as it is right now because it works with real data but I do not feel comfortable about how the specs are written because, myself, I have hard-time understanding what's going on.

It's not something that can be done with a one-man effort.